tests/pki.mk

```diff
@@ -0,0 +1,72 @@
+$(PKI_DIR):
+	mkdir -p $(PKI_DIR)
+
+	# create a certificate authority (CA) that both the client and server trust.
+	# The CA is just a public and private key with the public key wrapped up in a self-signed X.509 certificate.
+	openssl req \
+		-new \
+		-x509 \
+		-nodes \
+		-days 365 \
+		-subj '/C=GB/O=Example/OU=TeamA/CN=ca.example.com' \
+		-keyout $(PKI_DIR)/ca.key \
+		-out $(PKI_DIR)/ca.crt
+
+	# create the server’s key
+	openssl genrsa \
+		-out $(PKI_DIR)/server.key 2048
+
+	# create a server Certificate Signing Request
```

Comment:
This step fails with the following error:

```
mkdir -p ./grpc/pki
# create a certificate authority (CA) that both the client and server trust.
# The CA is just a public and private key with the public key wrapped up in a self-signed X.509 certificate.
openssl req \
	-new \
	-x509 \
	-nodes \
	-days 365 \
	-subj '/C=GB/O=Example/OU=TeamA/CN=ca.example.com' \
	-keyout ./grpc/pki/ca.key \
	-out ./grpc/pki/ca.crt
3m54s234ms
Generating a RSA private key
..........................................+++++
...+++++
writing new private key to './grpc/pki/ca.key'
-----
unable to find 'distinguished_name' in config
problems making Certificate Request
139874421728576:error:0E06D06C:configuration file routines:NCONF_get_string:no value:../crypto/conf/conf_lib.c:273:group=req name=distinguished_name
make: *** [pki.mk:5: grpc/pki] Error 1
```

Could you handle this ?

Reply:
Is the CI on linux?

Reply:
yes, ubuntu-20.04

Reply:
I quickly searched and found this: https://superuser.com/questions/947061/openssl-unable-to-find-distinguished-name-in-config
It looks like `openssl req` is not able to find a config value...
could you please let me know which openSSL version you run on ubuntu?

Reply:
Also, we don't use `-subj` in test, so perhaps that could be removed and see if the `distinguished_name` issue goes away

Reply:
I have tried to:
1. Run in Docker ubuntu:20.04
2. Install openssl with `apt update && apt install --yes openssl`
3. Run 2 commands and they were successful:
```
root@7bd2261de939:~# mkdir -p ./grpc/pki
root@7bd2261de939:~# openssl req \
> -new \
> -x509 \
> -nodes \
> -days 365 \
> -subj '/C=GB/O=Example/OU=TeamA/CN=ca.example.com' \
> -keyout ./grpc/pki/ca.key \
> -out ./grpc/pki/ca.crt
Generating a RSA private key
.........................+++++
...................................................................................................................+++++
writing new private key to './grpc/pki/ca.key'
-----
```
Question would be: Are you using vanilla Ubuntu 20.04 Docker image? Does it have openssl installed ? Or you use some other custom Docker image with openssl installed in some other way ?

Reply:
It seems your openssl can't find `/usr/lib/ssl/openssl.cnf` thus question is - is this file available on your Docker image?

Reply:
@sarunask @fabriziomoscon  openssl fixed on your worker model. Tests are now ok. Can you rebase your PR please?

Reply:
done